problems/5.loops.js

```diff
@@ -2,9 +2,16 @@
 * @param {number} num - The number to iterate up to
 * @param {number[]} - An array of even numbers from 0 to {num}
 */
-function evenNumbers() {
+function evenNumbers(num) {
+  let arr = [];
+  for(i = 0; i < num.length; i++) {
```

Comment:
num is a number not an array. That's like saying 5.length -> not a thing in javascript. Try to break things down into their data types. i can also be incremented by 2. i += 2